### ISSUE TITLE
RxMap onChange Listener

### ISSUE THREAD

**Issue by KDCinfo** [Jaguar-dart/observable_ish#8]
How does one listen for changes for an `RxMap`?

[The docs say](https://pub.dev/documentation/observable_ish/latest/)

> Listening for changes: `onChange` exposes a `Stream` of record of change of the `Map`.

But [the RxMap Class docs](https://pub.dev/documentation/observable_ish/latest/map_map/RxMap-class.html) don't show an `onChange`, and my editor shows: 
```
	"code": "undefined_getter",
	"message": "The getter 'onChange' isn't defined for the type 'RxMap<dynamic, dynamic>'.\nTry importing the library that defines 'onChange', correcting the name to the name of an existing getter, or defining a getter or field named 'onChange'."
```
I just updated to version 3, but I couldn't figure this out on version 2 either.

Thanks for any insights.

**Comment by KDCinfo** [comment]
Can we use an `RxSet` in lieu of an `RxMap`?

**Comment by tejainece** [comment]
This is now available in 4.0.0.

Thanks for the PR.
